hack/util.sh

```diff
@@ -102,6 +102,19 @@ function util::create_certkey {
 EOF
 }
 
```

Comment:
add more details about the function would make it easier for people to understand

Reply:
done